code/game/objects/items/weapons/weaponry.dm

```diff
@@ -278,7 +278,7 @@
 	desc = "A chainsaw that has replaced your arm."
 	icon_state = "chainsaw_on"
 	item_state = "mounted_chainsaw"
-	flags = ABSTRACT | NODROP
+	flags = NODROP

```

Comment:
I'm a bit worried about getting rid of the `ABSTRACT` here but I think `NODROP` already handles the edge cases we want pretty well. We can always revert it back if something breaks, after all.